src/components/WalletDropdown/SlideOutMenu.tsx

```diff
@@ -0,0 +1,62 @@
+import { ChevronLeft } from 'react-feather'
+import { Text } from 'rebass'
+import styled, { useTheme } from 'styled-components/macro'
+
+const BackSection = styled.div`
+  position: relative;
+  display: flex;
+  padding: 0 16px;
+  color: ${({ theme }) => theme.textTertiary};
+  cursor: default;
+  :hover {
+    text-decoration: none;
+  }
+`
+
+const Menu = styled.div`
+  width: 100%;
+  height: 100%;
+  font-size: 16px;
+  overflow-y: scroll;
+  &::-webkit-scrollbar {
+    display: none;
+  }
+`
+
+export const SlideOutMenu = ({
+  children,
+  close,
+  title,
+  clear,
+}: {
+  close: () => void
+  title: JSX.Element
+  children: JSX.Element | Array<JSX.Element>
+  clear?: () => void
```

Comment:
similar to above, `onClear`? it's actually unclear what difference between clear and close is just by looking at the prop names.